CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 ###############################################################################
 # Minimum CMake version and policies
 ###############################################################################
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
```

Comment:
I would go then for upgrading both as well, no? I mean, I don't suspect developers to use a three year old distro release on their development machine.